dev-packages/rollup-utils/plugins/bundlePlugins.mjs

```diff
@@ -63,8 +63,9 @@ export function makeIsDebugBuildPlugin(includeDebugging) {
 export function makeSetSDKSourcePlugin(sdkSource) {
   return replace({
     preventAssignment: false,
+    delimiters: ['', ''],
```

Comment:
So with this change, we should now have no real bundle size impact in either scenario. We simply do not overwrite stuff at all by default, returning `npm`. But for CDN bundles, we replace the _comment_, not the value itself, which is correctly minified to what we want.

Reply:
Note that without the delimiter part, it does not correctly replace the string, for whatever reason.